packages/types-android/src/lib/android.d.ts

```diff
@@ -1 +1 @@
-/// <reference path="./android-17.d.ts" />
\ No newline at end of file
+/// <reference path="./android-33.d.ts" />
```

Comment:
Per @rigor789 mention, please use 30 as it's the current recommendation from Google.
```suggestion
/// <reference path="./android-30.d.ts" />
```